projects/plugins/jetpack/extensions/plugins/post-publish-qr-post-panel/index.js

```diff
@@ -1,9 +1,13 @@
 /**
  * External dependencies
  */
+import React from 'react';
 import { PluginPostPublishPanel } from '@wordpress/edit-post';
 import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
 import { SVG, Path } from '@wordpress/components';
```

Comment:
We should be able to not import React here, since we're inside the editor context.

```suggestion
import { PluginPostPublishPanel } from '@wordpress/edit-post';
import { __ } from '@wordpress/i18n';
import { useSelect } from '@wordpress/data';
import { store as editorStore } from '@wordpress/editor';
import { SVG, Path, Component } from '@wordpress/components';
```